src/Datadog.Trace/AgentHttpHeaderNames.cs

```diff
@@ -37,5 +37,10 @@ internal static class AgentHttpHeaderNames
         /// The id of the container where the traced application is running.
         /// </summary>
         public const string ContainerId = "Datadog-Container-ID";
+
+        /// <summary>
+        /// The resource id of the site instance in azure app services where the traced application is running.
+        /// </summary>
+        public const string AzureAppServicesResourceId = "Datadog-AzureAppServices-Resource-ID";
```

Comment:
This key is actually yet to be determined. Meeting with the backend team and Garner to discuss.